stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -564,112 +559,19 @@ public class ChannelController internal constructor(
         return response
     }
 
-    /**
-     * - Generate an ID
-     * - Insert the message into offline storage with sync status set to Sync Needed
-     * - If we're online do the send message request
-     * - If the request fails we retry according to the retry policy set on the repo
-     */
-    @Deprecated(
-        message = "Don't use sendMessage with attachmentTransformer. It's better to implement custom attachment uploading mechanism for additional transformation",
```

Comment:
Not sure what this "custom attachment uploading mechanism" should look like, or how it would be added by clients